### DISCUSSION TITLE
Following a link in a docs popup

### DISCUSSION THREAD

**Discussion by LordMZTE** [hrsh7th/nvim-cmp#1799] [Q&A]
Some LSPs provide links in doc popups that can be clicked, here's an example with a popup provided by ZLS:
![screenshot](https://github.com/hrsh7th/nvim-cmp/assets/28735087/0c62816d-6eca-4a55-8ae4-7bad790973bc)
In VSCode, you'd be able to click on "Allocator" here (at the bottom of the popup). How can I set a keybind to follow this link in CMP?

**Comment by Mr-Bojangles** [comment]
I'm curious if this is possible, as well?  It would be nice if I could do something like <Shift-k> to get into it, like I do with LSP hovering.
